web/html/src/manager/shared/spa/spa-engine.js

```diff
@@ -37,6 +37,12 @@ window.pageRenderers.spa.init = function init() {
       `;
       window.location = urlPath;
     }
+
+    // If an error happens we make a full refresh to make sure the original request is shown instead of a SPA replacement
+    if (navigation.error) {
+      window.location = navigation.path;
```

Comment:
Is this error caught by the normal browser event `on error`? I'd really like to have this logged into `rhn_web_frontend.log` file so if the `error` event is not thrown automatically, I'd add log this error manually anyway.

Reply:
This is all the errors apart from >200 <400. Senna doesn't know how to handle them, so it's better to just redirect to the initial URL. They were already logged